### ISSUE TITLE
Possible enhancements

### ISSUE THREAD

**Issue by TehPeGaSuS** [supakeen/pinnwand#245]
Here's some ideas about possible enhancements to pinnwand:
- [ ] Add `Burn after reading`, in case we want to keep the post longer but to be deleted right after the person read it.
- [ ] Add an option like `expiries_default = 3600` or `expiries_default = ["expiries.1day"]` so we can select what the default expiry is and still have a selectable range of expiries, pretty much like the `preferred_lexers` option.
- [ ] Add an option to upload files, such as images/videos, with the correspondent option to limit file size.

**Comment by supakeen** [comment]
Going through them.

1. Can't be guaranteed due to caching but might be nice-ish.
2. Other programs and websites exist for this, it's a difficult battle with illegal and copyrighted content which most public instances of pastebins already deal with on just text.
3. Agreed. Let's add that.

**Comment by realies** [comment]
i wanted to use pinnwand for occasional small file uploads but after the browser tab froze a couple of times i found out the file content gets inserted into the textarea... can we fix this so that it works as expected please?

**Comment by supakeen** [comment]
@realies How would you want it to work instead? I don't really know what the/your expectation is :)

**Comment by realies** [comment]
when i see click/drag here to upload i expect it to add the file as an attachment to the paste entry, not to use the textarea for its content

**Comment by supakeen** [comment]
Right, so see the above reasoning for why I'd really not like to accept 'attachments' or non-text content in `pinnwand` or did you mean that it would still show up as a text file *after* pasting?

How can I make it more clear that dragging/dropping (or uploading) a file causes a new text file to be added?

Also the 'hang when drag/drop' smells like a bug and probably deserves a separate ticket. That sounds very not nice to have.

**Comment by realies** [comment]
make attachments optional would be best in my opinion; the hanging is probably due to loading a multi-megabyte binary file inside a textarea
